src/panels/config/devices/device-detail/ha-device-info-card.ts

```diff
@@ -135,6 +138,10 @@ export class HaDeviceCard extends LitElement {
         color: var(--secondary-text-color);
         word-wrap: break-word;
       }
+      a {
```

Comment:
Weird this was necessary

Reply:
Dunno either, It looked weird/of without it.